include/boost/beast/http/fields.hpp

```diff
@@ -435,12 +444,14 @@ class basic_fields
         @param name The field name.
 
         @param value The value of the field, as a @ref boost::beast::string_view
+
+        @throws boost::system::system_error Thrown if an error occurs:
+        - If the size of @c value exceeds @ref max_value_size, the
+            error code will be @ref error::header_field_value_too_large.
     */
     void
     insert(field name, string_view const& value);
```

Comment:
string views are typically passed by value

Reply:
Having both styles in the same parameter list is weird... I think we should change them in this file.